src/matrix_view.py

```diff
@@ -0,0 +1,70 @@
+import ast
+
+from src.matrix_operation import MatrixOperation
+
+
+class MatrixView:
+    def __init__(self, controller):
+        self._controller = controller
+        self._selected_operation = None
+        self._matrix1 = None
+        self._matrix2 = None
+
+    def start(self):
+        self._selected_operation = MatrixView.select_operation()
+        self.create_matrix(self._selected_operation)
+        self.perform_operation(self._matrix1, self._matrix2, self._selected_operation)
+
+    @staticmethod
+    def select_operation():
+        while True:
+            user_input = input(
+                "Enter the number corresponding to a matrix operation:\n "
+                "1 - Addition, "
+                "2 - Multiplication, "
+                "3 - Inverse:\n "
+            )
+            try:
+                user_input = int(user_input)
+                if user_input in [1, 2, 3]:
+                    return MatrixOperation(user_input)
+                    break
+                else:
+                    print("Enter a value between 1 and 3")
+            except ValueError:
+                print("Value error")
+
+    def create_matrix(self, matrix_operation):
+        if matrix_operation == MatrixOperation.ADDITION:
+            self._create_addition_matrix()
```

Comment:
Rather than a large if .. elif construct you could use a collection of function references and index in (after sanitising user input) instead.
Something like:
```python
self.operations = {
MatrixOperation.ADDITION: create_addition_matrix,
MatrixOperation.MULTIPLICATION: create_multiplication_matrix,
}
```

Reply:
Fixed